### PR TITLE
update project readme

### PR DESCRIPTION
**Pull request by ambid17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,41 @@
 # BunnyHopMaster
-A 3D platformer based on the Source engine movement mechanics, portals, and a basis in speedrunning
+JumpingJax is a 3d platformer video game make using the Unity game engine.
 
-Unity Version: 2019.3.0f3
+The core idea of the game is that the player has tools such as portals and anti-friction boots to find their lost dog.
 
-![Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/giphy.gif)
+![Game Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/jumpingjax.gif)
```
Comment (0xLeif) [inline README.md]: Nothing here
Game Preview

**Review by 0xLeif — CHANGES_REQUESTED** [review]
Fix Game Preview

**Comment by 0xLeif** [comment]
This should go `develop` then `master`

File: README.md [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
-* Teleport a player
-* Allow the player to see a new location through the portal
 
 ****
-# <Project Name>
-## oneleif Active Project
+## oneleif project
+This means that the project is sponsored by the oneleif community, and the collaborators are team members from oneleif.
 
 ![](https://github.com/oneleif/olDocs/blob/master/assets/images/oneleif_logos/full_logo/oneleif_whiteback.png)
```
Comment (Vextin) [inline README.md]: Link to the oneleif website when the user clicks the banner. Syntax [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos) 

```suggestion
![](https://github.com/oneleif/olDocs/blob/master/assets/images/oneleif_logos/full_logo/oneleif_whiteback.png)(https://oneleif.com)
```

File: README.md [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
 [![](https://img.shields.io/badge/oneleif-Twitter-blue.svg)](https://twitter.com/oneleifdev)
 
 [![](https://img.shields.io/badge/oneleif-YouTube-red.svg)](https://www.youtube.com/channel/UC3HN0jID38K0Vb_WChvgQmA)
 
-## How to join oneleif
-Click on the link below to join the Discord server.
-
-You will start with limited permissions, in a text channel that only moderators will see.
+## What is oneleif?
+oneleif is a nonprofit community that supports tech minded individuals. We do this by offering a fun loving community that works on Open Sourced projects together.
```
Comment (Vextin) [inline README.md:25]: Writing style changes for conciseness and clarity 

```suggestion
oneleif is a nonprofit community that supports tech minded individuals. We offer a fun loving community that works on open-source projects together. 
```

**Review by Vextin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ambid17) [reply README.md]: updated

Comment (0xLeif) [reply README.md]: Doesn’t seem to work on mobile

**Review by Vextin — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,42 +1,40 @@
 # BunnyHopMaster
-A 3D platformer based on the Source engine movement mechanics, portals, and a basis in speedrunning
+JumpingJax is a 3d platformer video game make using the Unity game engine.
 
-Unity Version: 2019.3.0f3
+The core idea of the game is that the player has tools such as portals and anti-friction boots to find their lost dog.
 
-![Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/giphy.gif)
+[![Watch the video](https://media.giphy.com/media/SWhvI3prm9w6NFstn0/giphy.gif)]
```
Comment (0xLeif) [inline README.md]: Watch the video

File: README.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ JumpingJax is a 3d platformer video game make using the Unity game engine.
 
 The core idea of the game is that the player has tools such as portals and anti-friction boots to find their lost dog.
 
-![Game Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/jumpingjax.mp4)
+[![Watch the video](https://media.giphy.com/media/SWhvI3prm9w6NFstn0/giphy.gif)]
```
Comment (0xLeif) [inline README.md]: You don't need the `[]`

just `Watch the video`

**Review by 0xLeif — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,42 +1,40 @@
...
-* Teleport a player
-* Allow the player to see a new location through the portal
 
 ****
-# <Project Name>
-## oneleif Active Project
+## oneleif project
+This means that the project is sponsored by the oneleif community, and the collaborators are team members from oneleif.
+
+![](https://github.com/oneleif/olDocs/blob/master/assets/images/oneleif_logos/full_logo/oneleif_whiteback.png)(https://oneleif.com)
```
Comment (0xLeif) [inline README.md]: [](https://oneleif.com)

**Review by 0xLeif — DISMISSED** [review]

**Review by 0xLeif — APPROVED** [review]

**Review by Vextin — APPROVED** [review]

**Pull request merged**